core/src/main/kotlin/org/evomaster/core/search/gene/Gene.kt

```diff
@@ -50,6 +50,11 @@ abstract class Gene(var name: String) {
      */
     abstract fun copy() : Gene
 
+    /**
+     * weight for mutation
```

Comment:
add bit more info: eg, higher the weight, the higher the chances to be selected for mutation